### ISSUE TITLE
superfluous local in AMP.__init__

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3301]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3301|
|Type|defect|
|Created|2008-06-20 18:36:16Z|

http://twistedmatrix.com/trac/browser/trunk/twisted/protocols/amp.py?rev=22230#L2055

Attachments:

* [amp.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/453/453c4ef6c6367eeaf55e07aa78e53967b96d7be6/2d9a359c76c27e35118c98683eda4f2c0de18fd8.patch) (431 bytes) - added by exarkun on 2008-07-03 11:37:18Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3301 3301
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1213986976000000 1213986976000000
changetime__1215466625000000 1215466625000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="jkakar's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|jkakar commented|
|-|-|

I've reviewed the change and it looks good.  +1!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [24238]) Apply amp.patch, removing unneeded code

Author: exarkun
Reviewer: jkakar
Fixes: [#3301](https://github.com/twisted/twisted/issues/3301)

`AMP.__init__` does not need to create a `boxSender` local since it does not
use this local anywhere.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
